src/test/java/edu/unc/lib/boxc/migration/cdm/services/GroupMappingServiceTest.java

```diff
@@ -446,11 +417,19 @@ private void assertParentIdsPresent(Connection conn, String... expectedParents)
         while (rs.next()) {
             parentIds.add(rs.getString(1));
         }
-        List<String> expected = Arrays.asList(expectedParents);
+        var expected = Arrays.stream(expectedParents).map(this::asGroupKey).collect(Collectors.toList());
         assertTrue("Expected parent ids " + expected + " but found " + parentIds, parentIds.containsAll(expected));
         assertEquals("Expected parent ids " + expected + " but found " + parentIds, expected.size(), parentIds.size());
     }
 
+    private String asGroupKey(String matchValue) {
```

Comment:
I think I kind of like it better without the else. like just the if block and then the return? also i don't quite get what this method is doing from the name...can you explain?

Reply:
I've added javadoc to the method. Stylistically, I see the branches in the conditional here as being fairly equal rather than an early return, so I think i prefer keeping the if+else, but don't have strong feelings about it